src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -110,7 +110,8 @@ public void Initialize()
                 : StorageEnvironmentOptions.ForPath(System.IO.Path.Combine(Configuration.Core.DataDirectory,"System"));
 
             options.SchemaVersion = 2;
-
+            options.RunWithSparseMmapPagerConfiguration = Configuration.Core.RunWithSparseMmapPager;
```

Comment:
This isn't the only place this is used.
In fact, you need to make sure that this is handled everywhere.
Including in the `DatabaseLandlord`, `SubscriptionStorage`, `Configuration`, `Indexes`, etc.